backend/src/main/java/ubb/courses/backend/models/Course.java

```diff
@@ -44,4 +43,7 @@
     @ManyToOne()
     @JoinColumn(name = "owner_id",referencedColumnName = "id")
     private User ownerId;
+
+    @OneToMany(mappedBy = "course")
+    private List<Media> media;
```

Comment:
> When you need a set and never care about order, use a Set. When for some reason order is important (ordered list, ordering by date, etc.), then a List.

In our case make sure the media is returned by repository ordered by orderIndex, otherwise use a set.